Plugins/RGMPlugin.h

```diff
@@ -12,13 +15,14 @@ class RGMPlugin : public QObject {
   ~RGMPlugin();
 
  signals:
-  void OutputRead(const QString &output);
-  void ErrorRead(const QString &error);
+  void LogOutput(const QString &output);

```

Comment:
Isn't this supposed to have a severity attached? I'll assume you haven't gotten to that, yet.

Reply:
I haven't gotten to it yet 😋